src/main/java/org/example/C02_20_MultipleOfNumbers.java

```diff
@@ -2,13 +2,13 @@
 
 import java.util.Scanner;
 
-public class C02_20_MultipleOf {
+public class C02_20_MultipleOfNumbers {
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         System.out.println("Enter your number: ");
         int number = scanner.nextInt();
 
-        if (number % 3 == 0 ^ number % 7 == 0) {
+        if (number % 3 == 0 || number % 7 == 0) {
```

Comment:
The problem says "if it is multiple of both 3 and 7 or neither of them will display false". I dont think your solution respects that.